search/includes/classes/class-queue.php

```diff
@@ -266,13 +266,26 @@ public function count_jobs( $status, $object_type = 'post' ) {
 
 		$table_name = $this->schema->get_table_name();
 
-		return $wpdb->get_var(
-			$wpdb->prepare(
-				"SELECT COUNT(*) FROM {$table_name} WHERE `status` = %s AND `object_type` = %s", // Cannot prepare table name. @codingStandardsIgnoreLine
-				$status,
-				$object_type
-			)
+		$query = $wpdb->prepare(
+			"SELECT COUNT(*) FROM {$table_name} WHERE `status` = %s AND `object_type` = %s", // Cannot prepare table name. @codingStandardsIgnoreLine
+			$status,
+			$object_type
 		);
+
+		if ( 'all' === strtolower( $status ) ) {
```

Comment:
Will there be a case where we want to query all object types, but one specific status? Such as to see how many queued items there are across all types.

Reply:
Not necessarily worth changing for this, just noting it :)